src/cpu/cpu.cpp

```diff
@@ -158,16 +158,6 @@ Bitu CPU_Pop32(void) {
 	return val;
 }
 
-PhysPt SelBase(Bitu sel) {
-	if (cpu.cr0 & CR0_PROTECTION) {
-		Descriptor desc;
-		cpu.gdt.GetDescriptor(sel,desc);
-		return desc.GetBase();
-	} else {
-		return sel<<4;
-	}
-}
-
```

Comment:
Did you mean to completely remove this function? If so, the commit name is misleading.

Reply:
Yes, it's was added in 2003 but never used, so has sat as a vestigial remnant. 
I've renamed the commit message to clearer: _"Remove an unused function to eliminate an unchecked rcode warning"_